html_css_exercise/femc_level1_huddleLandingPage/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="css/index.css" />
+        <link rel="stylesheet" href="css/reset.css" />
```

Comment:
Your reset should be linked first, before your index. Order is important here. CSS imported later will override CSS imported earlier. You want to start with your reset to get a clean slate, then override from there with your own custom styles. You don't want the reset overriding your custom styles. 

```suggestion
        <link rel="stylesheet" href="css/reset.css" />
        <link rel="stylesheet" href="css/index.css" />
```